src/java/fr/paris/lutece/plugins/limitconnectedusers/service/filter/LimitConnectedUsersFilter.java

```diff
@@ -187,7 +193,7 @@ public void init( FilterConfig filterConfig )
             _bActivate = new Boolean( paramValue );
         }
 
-        _nMaxConnectedUsers = AppPropertiesService.getPropertyInt( PROPERTY_MAX_CONNECTED_USERS, DEFAULT_NB_MAX );
+        _nMaxConnectedUsers = (int) AppPropertiesService.getPropertyLong( PROPERTY_MAX_CONNECTED_USERS, DEFAULT_NB_MAX );

```

Comment:
what's this ?

Reply:
AppPropertiesService.getPropertyInt() does not recognize negative int !